source/vibe/http/common.d

```diff
@@ -426,11 +426,13 @@ final class ChunkedInputStream : InputStream {
 	Outputs data to an output stream in HTTP chunked format.
 */
 final class ChunkedOutputStream : OutputStream {
+	alias ChunkExtensionCallback = string delegate(in ubyte[] data);
```

Comment:
Note that I changed the delegate's signature to taking an argument of `in ubyte[]` instead of `const(ubyte)[]`. This should prove to be especially useful for a `VibeManualMemoryManagement` version later on (and, in fact, already is now, since one `write()` overload also takes `in ubyte[]`).